spec/integration/wiki_spec.js

```diff
@@ -0,0 +1,496 @@
+              User.findOne({where: {id: user.id}}).then(() => {
+    
+                request.post(`${accountBase}upgrade`, (err, res, body) => {
+                  User.findOne({where: {id: user.id}}).then((premiumdUser) => {
+                    this.premiumUser = premiumdUser;
+                    
+                    Wiki.create({
+                        title: "Wizard of Oz",
+                        body: "Everything is better in green!",
+                        userId: this.premiumUser.id,
+                    }).then((wiki) => {
+                        this.privateWiki = wiki;
+                    }).catch((err) => {
+                        console.log(err);
+                        done();
+                    })
+                  }).catch((err) => {
+                    console.log(err);
+                    done();
+                  })             
+              });
+            }).catch((err) => {
+              console.log(err);
+              done();
+            });   
+          }).catch((err) => {
+            console.log(err);
+            done();
+          })
+        });
+    
+
+        describe("GET /wikis", () => {
+
+            it("should render a view with all public wikis", (done) => {
+                request.get(base, (err, res, body) => {
+                    Wiki.findAll({where: {private: false}}).then(() => {
+                        expect(body).toContain("Wiki Index");
+                        expect(body).toContain("Dorothy in Oz");
+                        done();
+                    })
+                    .catch((err) => {
+                        console.log(err);
+                        done();
+                    })
+                });
+
+            });
+
+            it("should render a view with any private wikis that are owned by the user", (done) => {
+                Wiki.create({
+                    title: "Bogs and Swamps",
+                    body: "Swampy bogs and boggy swamps!",
+                    private: true,
+                    userId: this.premiumUser.id
+                }).then((privateWiki) => {
+                    this.newWiki = privateWiki;
+
+                    request.get(base, (err, res, body) => {
+
+                        Wiki.findAll({where: {userId: this.premiumUser.id}}).then((wikis) => {
+                            expect(body).toContain("Bogs and Swamps");
+                            expect(body).toContain("This is a Private Wiki");
+                            done();
+                        })
+                        .catch((err) => {
+                            console.log(err);
+                            done();
+                        })
+                    });
+                })
+                .catch((err) => {
+                    console.log(err);
+                    done();
+                });
+
+            });
+
+            it("should respond with all private wikis where the user is a collaborator", (done) => {
```

Comment:
For a lot of these tests you should have an additional test for testing the negative i.e "it should NOT respond with private wikis where the user is NOT a collaborator"